citrix_hypervisor/README.md

```diff
@@ -0,0 +1,59 @@
+# Agent Check: citrix_hypervisor
+
+## Overview
+
+This check monitors [citrix_hypervisor][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The citrix_hypervisor check is included in the [Datadog Agent][2] package.
```

Comment:
```suggestion
The Citrix Hypervisor check is included in the [Datadog Agent][2] package.
```

Reply:
this should probably get fixed in the ddev scaffolding template